crates/cli-support/src/js/mod.rs

```diff
@@ -1735,7 +1734,7 @@ wasm = wasmInstance.exports;
         if !self.should_write_global("symbol_dispose") {
             return Ok(());
         }
-        self.global("if(!Symbol.dispose) { Symbol.dispose = Symbol('Symbol.dispose'); }");
+        self.global("const symbolDispose = Symbol.dispose || Symbol.for('wbg_symbol_dispose');");
```

Comment:
This is what I was confused about: why do we need this in the first place?
We should be able to just use:
```js
    [Symbol.dispose]() {
        this.free();
    }
```
Regardless of support. It shouldn't trigger an error and we aren't overwriting anything either.

I'm suggesting:

```suggestion
```

Reply:
We wouldn't be able to make it the default since `Symbol.dispose` doesn't exist in Safari and would be `undefined` defining `MyClass.prototype.undefiend = ....`. https://caniuse.com/mdn-javascript_builtins_symbol_dispose

Reply:
What do you mean with the "default" here? We have to keep `free()` anyway, so that's the default (if that is what you meant).

Reply:
I mean defaulting to enabling the Symbol.dispose output, whereas right now it is still flagged.

Reply:
So whats wrong with `MyClass.prototype.undefined = ...`? From me trying it out it seems to not cause any errors.

Reply:
Oh ... I get it now. I thought in this case it would just not set anything at all. Instead it actually sets it on a field called `undefined`. That explains why it doesn't error.

What do you think of just defining it conditionally then?
```js
static {
    if (Symbol.dispose) {
        this.prototype[Symbol.dispose] = function() {
            this.free();
        };
    }
}
```

Reply:
Well, `free()` is still available and we can't remove it anyway until the next breaking change. So I don't think there is a reason to introduce another fallback?

Reply:
Yeah that makes sense sure.

Reply:
Added in the latest commit.